doc_admin/AdminManual.adoc

```diff
@@ -425,9 +425,9 @@ NOTE: Remember to review your setup in the Open Hospital configuration files and
 
 === 2.2 OH - PORTABLE mode
 
-Open Hospital is a Java software program and it is portable by definition, since it does not require installation. The *PORTABLE* mode refers to the possibility of deploying and executing a single instance of the OH software in a self-contained package that includes Java, MariaDB/MySQL Server and all the data. Once deployed on a single computer, the package can be copied and run everywhere - even from an USB stick - carrying together all the saved data.
+Open Hospital is a Java software program and it is portable by definition. The *PORTABLE* mode described in this chapter refers to the possibility of using OH as a self contained package that includes Java, MariaDB/MySQL Server and all the data, without requiring any software installation. The package can be used on a single computer and used run everywhere just by copying a folder - even onto an USB stick - carrying together all the saved data.
```

Comment:
fix the sentence  "... and used run everywhere just by copying a folder "

Reply:
Yes right :-) done !